WgWall/ClientApp/src/app/models/product.ts

```diff
@@ -1,6 +1,6 @@
 export class Product {
     id: Number;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this 'Number' wrapper object with primitive type 'number'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1533)